Formula/aichat.rb

```diff
@@ -0,0 +1,20 @@
+class Aichat < Formula
+  desc "ChatGPT cli"
```

Comment:
```suggestion
  desc "ChatGPT/GPT-3.5/GPT-4 in the terminal"
```

I would rephrase the description like that. Whatcha think?

Reply:
What if OpenAI releases more models in the future? Do we need to change every time a new model comes out?

Reply:
yeah, I think `ChatGPT cli` make sense, they probably gonna release a new version next year. :)